lib/rouge/lexers/msgtrans.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class MsgTrans < RegexLexer
+      title "MessageTrans"
+      desc "RISC OS message translator messages file"
+      tag 'msgtrans'
+      filenames 'Messages*'
+
+      state :root do
+        rule %r/^#[^\n]*/, Comment
+        rule %r/[^\n ,):?\/]+/, Name::Variable
+        rule %r/[\n\/?]/, Operator
+        rule %r/:/, Operator, :value
+      end
+
+      state :value do
+        rule %r/\n/, Text, :pop!
+        rule %r/%[0-3%]/, Operator
+        rule %r/./, Literal::String
```

Comment:
This will only match a single character at a time. That's less efficient than matching a longer series of character. To do that (assuming the earlier rules don't need modification), you could use` [^\n%]`.

Reply:
Good point, done.